assets/pages/contributions-landing-eu/contributionsLandingEUReducer.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { combineReducers } from 'redux';
+
+import { contributionSelectionReducerFor } from 'containerisableComponents/contributionSelection/contributionSelectionReducer';
+import { payPalContributionButtonReducerFor } from 'containerisableComponents/payPalContributionButton/payPalContributionButtonReducer';
+
+import type { State as ContributionSelectionState } from 'containerisableComponents/contributionSelection/contributionSelectionReducer';
+import type { State as PayPalState } from 'containerisableComponents/payPalContributionButton/payPalContributionButtonReducer';
+
+import type { CommonState } from 'helpers/page/page';
+
+
+// ----- Types ----- //
+
+type PageState = {
+  selection: ContributionSelectionState,
+  payPal: PayPalState,
+};
+
+export type State = {
+  common: CommonState,
+  page: PageState,
+};
+
+// ----- Initial State ----- //
+
+const initialStateOverrides = {
+  oneOffAmount: '50',
+  monthlyAmount: '10',
+  annualAmount: '75', // <-- this needs to DIE!
```

Comment:
Why this `this needs to DIE`? Is this logic implemented in contributions shared helper?

Reply:
It was because we don't offer annual uniformly. If that's not going to change anytime soon, it should probably go.